packages/validator/src/commands/validateCmd.ts

```diff
@@ -0,0 +1,7 @@
+import { validate } from '../validator';
+
+if (process.env.npm_config_specs || process.env.npm_config_template) {
+  console.log(validate(process.env.npm_config_specs, process.env.npm_config_template));
+} else {
+  console.log(validate(process.argv[3], process.argv[2]));
```

Comment:
Please add some comments explaining what the indices into `argv` refer to.

Reply:
Added in fb01cd9